src/utilities/readIndexConfig.js

```diff
@@ -0,0 +1,40 @@
+import fs from 'fs';
+import path from 'path';
+import hasIndex from './hasIndex';
+import {CREATE_INDEX_PATTERN} from './constants';
+
+const isString = (str) => {
```

Comment:
Remove unnecessary helpers. I am all pro declarative programming, though `typeof str === 'string'` is already achieving the same.